tests/python/frontend/onnx/test_forward.py

```diff
@@ -5077,7 +5077,7 @@ def verify_eyelike(indata):
     "test_reduce_sum_keepdims_random",
     "test_reduce_sum_negative_axes_keepdims_example",
     "test_reduce_sum_negative_axes_keepdims_random",
-    "test_reshape_allowzero_reordered",
+    # "test_reshape_allowzero_reordered",
```

Comment:
I think we probably want to just delete this line instead of commenting it out.

Reply:
bye bye test :wave: